eng/tools/rush-runner.js

```diff
@@ -59,7 +59,17 @@ const spawnNode = (cwd, ...args) => {
   console.log(`Executing: "node ${args.join(' ')}" in ${cwd}\n\n`);
   const proc = spawnSync('node', args, { cwd, stdio: 'inherit' });
   console.log(`\n\nNode process exited with code ${proc.status}`);
-}
+
+  if (proc.status !== 0) {
```

Comment:
We should probably just exit the runner script early if a subprocess returns a nonzero exit code:

```js
if (proc.status !== 0) {
  process.exit(proc.status || 1);
}
```

Reply:
Why?  Isn't it better to let all subprocesses run and use the last non-zero exit code?  This seems strictly better when running multiple subprocesses.

Reply:
I can see the value of it either way - exiting early if you know the step is going to fail saves time, but letting it run to completion can give you more context if there were multiple failures.

Reply:
If we just want to set the exit code, we can still clean it up as :
```js
process.exitCode = proc.status || 1;
```

Reply:
Do you mean this for the overall code change?

```js
if (proc.status !== 0) {
  // proc.status will be null if the subprocess terminated due to a signal, which I don't think
  // should ever happen, but if it does it's safer to fail.
  process.exitCode = proc.status || 1;
}
```

If so I agree this is simpler than the `if` block.